HW03/Alexey_Zalyotov/lib/person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :name, :surname
+  attr_accessor :notifications
+
+  def initialize(name:, surname:)
+    @name = name
+    @surname = surname
+    @notifications = []
+  end
+
+  def show_notifications
+    puts "____________________________\n" \
+        "#{self.class}`s notifications:"
+    notifications.empty? ? (puts 'No notifications') : (puts notifications)
+    puts
+  end
```

Comment:
this method has 2 main responsibilities: 
1) puts something 
2) prepare something 

Ideally, this method should looks like
```suggestion
  def show_notifications
    puts formated_notifications
  end
```

Reply:
I have trouble with this suggestion. In the method `show_notifications` I use `self.class`. I can`t access to right person class from Notification class.

Reply:
I came up with a solution!

Reply:
I mean just create a private method with this text 

```rb
  def show_notifications
    puts formated_notifications
  end

private

def formated_notifications
"____________________________\n" \
        "#{self.class}`s notifications:"
'No notifications'
...
end
```

Reply:
Done